dashboard/yarn.lock

```diff
@@ -910,9 +910,9 @@ babel-preset-jest@^20.0.3:
   dependencies:
```

Comment:
that's a lot of dependencies

Reply:
Yeah. The real top-level dependencies are stored in `dashboard/package.json.` Yarn generates a complete tree including all subdependencies with version numbers for consistency. I don't directly modify this file; it's auto-generated. They recommend to check it into version control.